src/pages/Stream/Views/Explore/StaticLogRow.tsx

```diff
@@ -27,21 +67,53 @@ const LogRow: FC<LogRowProps> = (props) => {
 		...headers.filter((header) => (isPinned ? !pinnedColumns.includes(header) : pinnedColumns.includes(header))),
 	];
 	const columnsToShow = headers.filter((header) => !columnsToIgnore.includes(header));
+
+	const handleMouseHover = useCallback((isCursorInside: boolean, row?: number, column?: number) => {
```

Comment:
This is still not correct. 
What if I send (true) as arg ?

Reply:
updated but please verify

Reply:
Instead of making args optional, create two separate functions for setHoveredCell(null) & setHoveredCell(index).